database/client.go

```diff
@@ -303,6 +303,29 @@ func (c *Client) GetPendingTxByHash(hash string) (*DBTx, error) {
 	return tx, err
 }
 
+//GetTotalTxz get row count of transaction table from mongo
+func (c *Client) GetTotalTxz() ([]*DBSimpleTxs, error) {
+	var DBSimpleTxs []*DBSimpleTxs
+	nTime := time.Now()
+	yesTime := nTime.AddDate(0, 0, -30)
+	logDay := yesTime.Format("2006-01-02")
+	query := func(c *mgo.Collection) error {
+		m := []bson.M{
+			{"$match": bson.M{"timetxs": bson.M{"$gte": logDay}}},
+			{"$group": bson.M{"_id": "$timetxs", "count": bson.M{"$sum": 1}}},
+			{"$sort": bson.M{"_id": -1}},
+		}
+		pipe := c.Pipe(m)
+		err1 := pipe.All(&DBSimpleTxs)
+		if err1 != nil {
+			return err1
+		}
+		return err1
```

Comment:
you can just merge the lines #319 ~ #323 into one statement:
return pipe.All(&DBSimpleTxs)